samples/ControlCatalog.NetCore.Native/ControlCatalog.NetCore.Native.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It could be better to reuse same old NetCore project but add msbuild property to enable/disable aot build.
What do you think?

Otherwise, this project should be excluded from dirs.proj, so it won't be built each time on CI. We don't need aot builds on CI currently.

Reply:
Will update the project to use msbuild property instead.

Reply:
Done.